triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -314,6 +324,12 @@ class Runner(
     (triggerMsgSource, postSubmitFailure)
   }
 
+  private def logReceivedMsg(tm: TriggerMsg): Unit = tm match {
+    case CompletionMsg(c) => logger.debug(s"trigger received completion message $c")
```

Comment:
```
20:58:20.100 [TriggerService-akka.actor.default-dispatcher-3] DEBUG com.daml.lf.engine.trigger.Runner - trigger received completion message Completion(658ee8d2-160f-413e-b6e3-c95ab01f4d7c,Some(Status(0,,Vector())),4,None) (context: {triggerDefinition=cc176cffd4667d83c42c1cba4402a21a24a05deb7ba6684cd1a7462809de72b1:TestTrigger:trigger, triggerId=535b9269-fe3f-426b-b83c-87b6c5b8b6c7})
```

Reply:
(Of course, you will not see `triggerId` in context with the non-service runner.)